manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -126,6 +126,12 @@ spec:
         - name: PROXY_CONFIG
           value: |
                  {{ protoToJSON .ProxyConfig }}
+        {{- if .ProxyConfig.ProxyMetadata }}
```

Comment:
Can you explain why do we need to support this in waypoint?

Reply:
this enable all features guard by env, e.g. PROXY_CONFIG_XDS_AGENT.

Reply:
was this missed at beginning

Reply:
@howardjohn is this by design?